### PR TITLE
[ADD] openacademy:  WebService. T#57454

### PR DESCRIPTION
**Pull request by ejdecena**
Exercise: Add a new service to the client.

### CODE REVIEW

File: docs/xmlrpc_client.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
+import xmlrpc.client
```
Comment (luisg123v) [inline docs/xmlrpc_client.py:1]: This change doesn't belong here

File: openacademy/models/openacademy_course.py [hunk 2]
```diff
@@ -10,8 +11,13 @@ class Course(models.Model):
     session_ids = fields.One2many("openacademy.session", string="Sessions", inverse_name="course_id")
 
     _sql_constraints = [
-        ("description_and_title_different", "CHECK(name != description)", "The title and description of the course must be different."),
-        ("unique_course_name", "UNIQUE(name)", "The name of the course must be unique.")
+        ("description_and_title_different",
```
Comment (luisg123v) [inline openacademy/models/openacademy_course.py:14]: Same here, this is an unrelated change

**Review by luisg123v — CHANGES_REQUESTED** [review]

File: docs/xmlrpc_client.py (hunk 1 above)
Comment (ejdecena) [reply docs/xmlrpc_client.py:1]: Before the `docs` folder was inside the module, isn't it correct that I removed it from the module?

File: openacademy/models/openacademy_course.py (hunk 2 above)
Comment (ejdecena) [reply openacademy/models/openacademy_course.py:14]: In the same PR where the REF was requested, these changes were also requested. I believe.

Comment (luisg123v) [reply openacademy/models/openacademy_course.py:14]: Yes, I asked both things, but I meant separated PRs for each of them.

Having changes separated is important to make commits granularized. In case one commit needs to be reverted, only changes that actually correspond to the reverted feature will be affected.

Comment (ejdecena) [reply openacademy/models/openacademy_course.py:14]: Okay, got it. I already made a new PR with requesting the REF -> https://github.com/ejdecena/openacademy/pull/12

**Review by luisg123v — APPROVED** [review]
